geometry_msgs/Point32 position  # position in the real world, position is the center of the of the patch
                                # the z value - if used - would be added to all values generated by the modify opertion
float32 orientation             # rotation angle of ellipse around z axis (measured in degrees)
sensor_msgs/Image patch         # an image representing the patch of terrain to update
string merge_method             # decides how to merge generated values with height values of the terrain
                                # available choices: { keep, replace, add, sub, min, max, avg }
                                # If not specified default is: add